### PR TITLE
update HA and postgresql configure doc

### PR DESCRIPTION
**Pull request by bybai**
for #4035 and #4075

1. enhance HA with share data using postgresql doc
2. add modify postgresql database directory doc

### CODE REVIEW

File: docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst [hunk 1]
```diff
@@ -80,7 +82,7 @@ Setup xCAT on the Primary Management Node
 
 #. Add the alias ip address into the ``/etc/resolv.conf`` as the nameserver. Change the hostname resolution order to be using ``/etc/hosts`` before using name server, change to "hosts: files dns" in ``/etc/nsswitch.conf``.
 
-#. Change hostname to the hostname that resolves to the Virtual IP address. This is required for xCAT and database to be setup properly.
+#. Change hostname to the hostname that resolves to the Virtual IP address, add the slias ip address and its hostname into the ``/etc/hosts``. This is required for xCAT and database to be setup properly.
```
Comment (robin2008) [inline docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst]: `slias` should be a typo.

File: docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst [hunk 2]
```diff
@@ -344,17 +357,23 @@ On the new primary management node:
...
+   * **[PostgreSQL]** ::
+
+    service postgresql start
+
 #. Start the daemons: ::
 
     service dhcpd start
     service xcatd start
-    service hdwr_svr start
     service conserver start
```
Comment (robin2008) [inline docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst]: need add it as optional when DFM is using.

File: docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst [hunk 3]
```diff
@@ -144,11 +158,11 @@ Setup xCAT on the Primary Management Node
 Setup xCAT on the Standby Management Node
 =========================================
 
-#. Make sure the standby management node is NOT using the shared data.
+#. Make sure the standby management node is NOT using the shared data.  
 
 #. Add the alias ip address into the ``/etc/resolv.conf`` as the nameserver. Change the hostname resolution order to be using ``/etc/hosts`` before using name server. Change "hosts: files dns" in /etc/nsswitch.conf.
 
-#. Temporarily change the hostname to the hostname that resolves to the Virtual IP address. This is required for xCAT and database to be setup properly. This only needs to be done one time.
+#. Temporarily change the hostname to the hostname that resolves to the Virtual IP address, add the slias ip address and its hostname into the ``/etc/hosts``. This is required for xCAT and database to be setup properly. This only needs to be done one time.
```
Comment (robin2008) [inline docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst]: using `Virtual IP address` here for `slias ip address`

File: docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst [hunk 4]
```diff
@@ -144,11 +158,11 @@ Setup xCAT on the Primary Management Node
 Setup xCAT on the Standby Management Node
 =========================================
 
-#. Make sure the standby management node is NOT using the shared data.
+#. Make sure the standby management node is NOT using the shared data.
```
Comment (robin2008) [inline docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst]: To remove the tailing space.

**Comment by robin2008** [comment]
Could you append your living UT doc link here?

**Comment by bybai** [comment]
@robin2008 , my UT link is ：http://10.4.41.1/install/pgsql/xcat-core/docs/build/html/advanced/hamn/setup_ha_mgmt_node_with_shared_data.html#setup-ha-mgmt-node-with-shared-data

**Pull request merged**